index.html

```diff
@@ -0,0 +1,216 @@
+                <img src="images/image.png" alt="A display">
+            </div>
+            <div class="work-info">
+                <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="languages">
+                    <li><a href="#">Ruby on rails</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">JavaScript</a></li>
+                    <li><a href="#">html</a></li>
+                </ul>
+                <button class="disp-project" type="button">
+                    <span class="project-title">See Project</span> </button>
+            </div>
+        </div>
+        <div class="wrk-container">
+            <div class="img-container">
+                <img src="images/image.png" alt="A display">
+            </div>
+            <div class="work-info">
+                <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="languages">
+                    <li><a href="#">Ruby on rails</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">JavaScript</a></li>
+                    <li><a href="#">html</a></li>
+                </ul>
+                <button class="disp-project" type="button">
+                    <span class="project-title">See Project</span> </button>
+            </div>
+        </div>
+        <div class="wrk-container">
+            <div class="img-container">
+                <img src="images/image.png" alt="A display">
+            </div>
+            <div class="work-info">
+                <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="languages">
+                    <li><a href="#">Ruby on rails</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">JavaScript</a></li>
+                    <li><a href="#">html</a></li>
+                </ul>
+                <button class="disp-project" type="button">
+                    <span class="project-title">See Project</span> </button>
+            </div>
+        </div>
+        <div class="wrk-container">
+            <div class="img-container">
+                <img src="images/image.png" alt="A display">
+            </div>
+            <div class="work-info">
+                <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="languages">
+                    <li><a href="#">Ruby on rails</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">JavaScript</a></li>
+                    <li><a href="#">html</a></li>
+                </ul>
+                <button class="disp-project" type="button">
+                    <span class="project-title">See Project</span> </button>
+            </div>
+        </div>
+    </section>
+    <section class="about">
+        <div class="about-container">
+            <h1 class="aboutme">About me</h1>
```

Comment:
Kindly center this header and also include the background image to match the Figma design.😉
![cent](https://user-images.githubusercontent.com/73854999/198722299-673b58ac-e1c8-40cf-87af-c58a5c6f7c22.PNG)

![inc](https://user-images.githubusercontent.com/73854999/198722689-9ddc42ad-dcff-450e-b4e1-273c7e10d9fb.PNG)

Reply:
- [ ] As the previous reviewer mentioned in this section, please add the background image to match the Figma design. I've seen that you are using the path on `./images` in lowercase, but your folder is Images starting with uppercase like `./Images` for the better practice you can change the name of your folder to start with a lowercase `i`. So that should be the problem on your code for sure, kindly change it and check its works before asking for a new review 🥇